<launch>
  <!-- 融合节点：先启动 -->
  <node name="fusion_node" pkg="offline_timestamp_align" type="fusion_node" output="screen"/>

  <!-- 发布节点：延迟1秒再启动,要不然同时启动会可能丢信息，在publish_node.cpp里发布逻辑和这里作双保险以保证信息传输的完整性 -->
  <node name="publish_node" pkg="offline_timestamp_align" type="publish_node" output="screen"
        launch-prefix="bash -c 'sleep 1; $0 $@' ">
    <param name="cam_csv" value="/home/cat/noetic_education_data/cam0_aligned.csv"/>
    <param name="image_dir" value="/home/cat/noetic_education_data/cam0/"/>
    <param name="imu_csv" value="/home/cat/noetic_education_data/imu0_data.csv"/>
  </node>
</launch>
